command/agent/host/host.go

```diff
@@ -53,8 +53,8 @@ func diskUsage(path string) (du DiskUsage, err error) {
 	used := disk - free
 	mb := float64(1048576)
 
-	disk = disk / mb
-	used = used / mb
+	disk /= mb
+	used /= mb
```

Comment:
I don't think I've ever seen this construct before (nor `*=`), so calling it "idiomatic" feels like the gocritic author's personal opinion leaching into a claim for "best practice". I don't have a _strong_ opinion about it but it feels weird and quirky rather than idiomatic. Maybe we leave this lint off?

Reply:
I also had never seen this before, and didn't even realize it was a thing. I'm happy to revert those changes.